### ISSUE TITLE
Multiple profiles issue in SMASH

### ISSUE THREAD

**Issue by ClaudevonRiegan** [yuzu-emu/yuzu#3645]
So if I create 2 or more profiles in yuzu smash ultimate softlocks on the loading screen itself on all profiles except the very first one.
     At the time of writing this i have yuzu 224 Mainline Build - 6d1ddafe1.
     I have attached photos for support too.
     I have renamed default yuzu profile as"aabhas".
     Rest 'aabhas2','aabhas3','aabhas4' i have created.
     So here 'aabhas' will work rest will softlock.
     If i delete 'aabhas' profile, then 'aabhas2' will work, 'aabhas3' and 'aabhas4' will softlock the 
     game.
     Basically the topmost profile works and rest others softlock in general.
![image](https://user-images.githubusercontent.com/63413745/79060305-18732680-7ca1-11ea-9620-8e955b335f72.png)
![Untitled](https://user-images.githubusercontent.com/63413745/79060369-de565480-7ca1-11ea-8801-0520ee65a2df.png)

Thank you Yuzu!!

**Comment by mariobro1233** [comment]
I have edited my issue after doing some more tests and this seems final.

**Comment by turkeynaruto** [comment]
Tested this myself on Yuzu Early Access 413 and it seems its true

**Comment by german77** [comment]
Fixed by #5842
